torchci/lib/drciUtils.ts

```diff
@@ -7,17 +7,17 @@ export const DOCS_URL = "https://docs-preview.pytorch.org/";
 export const PYTHON_DOCS_URL = "/index.html";
 export const CPP_DOCS_URL = "/cppdocs/index.html";
 export const DRCI_COMMENT_END = `\n
-This comment was automatically generated by Dr. CI.
+This comment was automatically generated by Dr. CI and updates every 15 minutes.
 <!-- drci-comment-end -->`;
 export const POSSIBLE_USERS = ["swang392"];
 export const HUD_URL = "https://hud.pytorch.org/pr/";
 
-export function formDrciComment(prNum: number): string {
-    let body = `## :link: Helpful Links
+export function formDrciHeader(prNum: number): string {
+    var body = `## :link: Helpful Links
 ### :test_tube: See artifacts and rendered test results [here](${HUD_URL}${prNum})
 * :page_facing_up: Preview [Python docs built from this PR](${DOCS_URL}${prNum}${PYTHON_DOCS_URL})
 * :page_facing_up: Preview [C++ docs built from this PR](${DOCS_URL}${prNum}${CPP_DOCS_URL})
 * :question: Need help or want to give feedback on the CI? Visit our [office hours](${OH_URL})
 Note: Links to docs will display an error until the docs builds have been completed.`;
-    return DRCI_COMMENT_START + body + DRCI_COMMENT_END;
+    return DRCI_COMMENT_START + body;
```

Comment:
nit: I think it would be clearer if you had a final method that was like
```
const comment = `
{DR_CI_COMMENT_START}
{body}
{pr_results}
{DR_CI_END}
`
```

or something like that

Reply:
agreed--this would be good